src/test/PdfTest/PdfMongoTests.java

```diff
@@ -0,0 +1,806 @@
+package PdfTest;
+
+import PDF.PdfController;
+import TestUtils.TestUtils;
+import kong.unirest.HttpResponse;
+import kong.unirest.Unirest;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.interactive.digitalsignature.PDSignature;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+// NOTE: Please delete file(s) uploaded at the end of your test or call clearAllDocuments() at the
+// beginning of a test case (right after logging in) to clear before running any assertions. Some
+// test cases rely on there only being one document in the database.
```

Comment:
you can use @BeforeClass and @AfterClass annotations to set these

Reply:
No, because it needs to be called for every test, not just once at the start

Reply:
then do @BeforeEach and @AfterEach but please never have code that you have to execute by reading comments

Reply:
Done